cibyl/utils/files.py

```diff
@@ -25,49 +26,44 @@ class FileSearch:
     """Allows for complex search queries targeting files on the filesystem.
     """
 
-    def __init__(self, directory):
+    def __init__(self, directory: str):
         """Constructor.
 
         :param directory: Path to directory to look for files in.
-        :type directory: str
         """
         self._directory = directory
         self._recursive = False
         self._extensions = []
         self._excluded = []
 
-    def with_recursion(self):
+    def with_recursion(self) -> 'FileSearch':
         """Extends the search to the folders inside the directory and beyond.
 
         :return: The instance.
-        :rtype: :class:`FileSearch`
         """
         self._recursive = True
         return self
 
-    def with_extension(self, extension):
+    def with_extension(self, extension: str) -> 'FileSearch':
         """Limits the search to files of a certain extensions. If this is
         called more than once, then the filters are joined together following
         and 'OR' approach.
 
         :param extension: The extension to filter by. Must be passed
             dot-prefixed, like: '.py'.
-        :type extension: str
         :return: The instance.
-        :rtype: :class:`FileSearch`
         """
         self._extensions.append(extension)
         return self
 
-    def with_excluded(self, excluded):
+    def with_excluded(self, excluded: list) -> 'FileSearch':
         """Limits the search to files that are not in the excluded list. If this
         is called more than once, then the filters are joined together
         following an 'OR' approach.
 
         :param excluded: The file names to filter by.
         :type excluded: list
```

Comment:
This should be removed also

Reply:
Changed. Thanks!

Reply:
I'm still seeing the line, maybe you forgot to push some commit?